ssn/chapters/CompleteExamples.html

```diff
@@ -0,0 +1,144 @@
+            corresponding to this example is available</a>. </p>
+    <pre class="example" data-include="./rdf/examples/apartment-134.ttl" data-include-format="text"></pre>
+</section>
+
+<section id="spinning-cups">
+    <h3>Wind sensor spinning cups</h3>
+    <p>An RDF file containing a <a
+            href="https://github.com/w3c/sdw/blob/gh-pages/sdw-sosa-ssn/ssn/rdf/examples/spinning-cups.ttl">graph
+            corresponding to this example is available</a>. </p>
+    <pre class="example" data-include="./rdf/examples/spinning-cups.ttl" data-include-format="text"></pre>
+</section>
+
+<section id="ice-core">
+    <h3>Ice Core</h3>
+    <p>In order to characterize a thing with a large extent, or which is not directly accessible, the usual
+        observational strategy is to obtain one or more samples. Observations can then be made more conveniently on
+        the samples, with the intention of characterizing the larger thing. This intentionality is captured using
+        the property <code>sosa:isSampleOf</code>. </p>
+    <p>In the following example, the ice core is a sample of the Antarctic ice sheet, and observations may be made on
+        the ice core and further samples taken from that. </p>
+    <p>A convenient side effect of this feature is that all observations related to the larger thing (the ice
+        sheet) can be found, and then potentially joined together in a meta-analysis in order to characterize that.
+    </p>
+    <p>An RDF file containing a <a
+            href="https://github.com/w3c/sdw/blob/gh-pages/sdw-sosa-ssn/ssn/rdf/examples/Sample-ice-core-bubble.ttl">graph
+            corresponding to this example is available</a>. </p>
+    <pre class="example" data-include="./rdf/examples/Sample-ice-core-bubble.ttl" data-include-format="text"></pre>
+</section>
+
+<section id="dht22-description">
+    <h3>Complex sensor capabilities &mdash; DHT22</h3>
+    <div class="ssn">
+        <p>An RDF file containing a <a
+                href="https://github.com/w3c/sdw/blob/gh-pages/sdw-sosa-ssn/ssn/rdf/examples/dht22.ttl">graph
+                corresponding to this example is available</a>. </p>
+        <pre class="example" data-include="./rdf/examples/dht22.ttl" data-include-format="text"></pre>
+    </div>
+</section>
+
+<section id="dht22-deployment">
+    <h3>DHT22 Deployment</h3>
+    <div class="ssn">
+        <p>This example shows how the conditions (temperature and humidity) in a room can be measured using one or
+            more sensors.
+            Each sensor observes the conditions in its immediate vicinity, and the values are then used to characterize
+            the room. </p>
+        <p>In Room 145, one of the walls is external in the building, so there is expected to be a temperature gradient
+            across the room, and there are two sensors on different walls.
+            In room 245, there is one sensor on the south wall.
```

Comment:
I cannot tell whether `Room 145` (capital `R`) or `room 145` (lowercase `r`) is desired. I'm guessing the former, in which case, `room 245` should become `Room 245`, as below. If my guess is wrong, then `Room 145` should become `room 145`.
```suggestion
        <p>In Room 145, one of the walls is external in the building, so there is expected to be a temperature gradient
            across the room, and there are two sensors on different walls.
            In Room 245, there is one sensor on the south wall.
```